src/validate.ts

```diff
@@ -1,9 +1,50 @@
 import R from 'ramda'
+import fs from 'fs'
+import path from 'path'
+import { findConfigFiles } from './utils/find-files'
+import { getFileFromPath } from './utils/get-file-from-path'
+import { validateConfig } from './utils/validate-config'
 
-export const validate = (schema: Schema): true => {
-  if (R.isNil(schema)) {
-    throw new Error('Config does not match the specified schema')
+const validateConfigAgainstSchema = (schema: Schema) => (
+  configFilePath: string
+): true => {
```

Comment:
maybe a stupid question, but shouldn't this be `boolean` instead of `true`?

Reply:
values can also be types, e.g. you can construct `type Levels = 'error | 'info' | 'debug'` as well. Since this function either throws or returns `true`, the stricter type here is the literal `true` rather than `boolean`.

Reply:
TIL, thx